src/SDK.ts

```diff
@@ -377,21 +416,28 @@ export default class SDK implements ISDK {
         const elapsedTimeInMilliseconds = timer.milliSecondsElapsed;
         if (this.logger) {
           await LoggingSanitizer.stripErrorSensitiveProperties(error);
-          this.logger.log(LogLevel.ERROR,
-            OCSDKTelemetryEvent.GETRECONNECTABLECHATS,
-            { ExceptionDetails: error, ElapsedTimeInMilliseconds: elapsedTimeInMilliseconds},
-            "Get Reconnectable Chats Failed");
+
+          const customData = {
```

Comment:
Missing `requestId`

Reply:
getReconnectableChats does not have requestId